source/docs/software/wpilib-tools/smartdashboard/stale-data-smartdashboard.rst

```diff
@@ -0,0 +1,8 @@
+Stale data and SmartDashboard
```

Comment:
This is a duplicate of https://docs.wpilib.org/en/latest/docs/software/wpilib-tools/smartdashboard/displaying-expressions.html?highlight=stale#stale-data

Reply:
Do you think it should live in its own article or in the `displaying-expressions` article?

Reply:
I'm not sure that it's big enough for it's own article

Reply:
I'll remove this article.